core/src/main/java/io/grpc/Status.java

```diff
@@ -391,12 +393,12 @@ public String parseAsciiString(String value) {
 
   /**
    * Extract an error {@link Status} from the causal chain of a {@link Throwable}.
-   * {@link Status#UNKNOWN} is returned if no status can be found or {@code t} is {@code null}.
+   * {@link Status#UNKNOWN} is returned if no status can be found.
```

Comment:
This isn't quite true, since Status.UNKNOWN is a particular instance. How about:
> If no status can be found, a Status is created with `t` as its cause.